datasetinsights/io/__init__.py

```diff
@@ -9,3 +13,23 @@
     "KubeflowPipelineWriter",
     "create_downloader",
 ]
+
+
+def gzip_decompress(filepath, destination):
+    """ Decompress a gzip file to the destination.
```

Comment:
Did you build the documentation to make sure this module is documented? Are the docstrings parsed properly?

Reply:
Removed this as it is not necessary right now.